pydm/widgets/drawing.py

```diff
@@ -1160,18 +1161,20 @@ def validate_point(i, point):
             """Ignore (instead of fail on) any of these pathologies."""
             if isinstance(point, str):
                 try:
-                    point = list(map(float, point.split(",")))
-                except ValueError:
+                    point = ast.literal_eval(point)
+                except SyntaxError:
                     logger.error("point %d must be two numbers, comma-separated, received '%s'", i, pt)
                     return
             if len(point) != 2:
```

Comment:
There probably should be a check above saying: `if not isinstance(point, (list, tuple))` just in case the user passes in, say, the value `5`

Reply:
@klauer I'll add the tests and make the code pass.

Reply:
done